various-receipes/3rd-party-services-isolation/test/isolation.test.js

```diff
@@ -0,0 +1,110 @@
+const request = require("supertest");
```

Comment:
What is the value in this test vs the basic examples? If we dedicated a recipe for isolation, maybe focus on simulation and show how to react to 3rd party corner cases: unusual response, errors, network down (!), etc

Reply:
I added 3 tests:
1. shows the basic isolation approach a.k.a interceptor, nock
2. shows an unusual response like 404. (maybe I should choose 500?)
3. how to mock 3rd party services with side effects like mailer/sms to prevent spam and so on.

What cases are missing?